CONTRIBUTING.rst

```diff
@@ -453,6 +453,29 @@ Step 4: Prepare PR
    PR guidelines described in `pull request guidelines <#pull-request-guidelines>`_.
    Create Pull Request! Make yourself ready for the discussion!
 
+5. The ``static checks`` and ``tests`` in your PR serve as a first-line-of-check, whether the PR
+   passes the quality bar for Airflow. It basically means that until you get your PR green, it is not
+   likely to get reviewed by committers unless you specifically ask for it and explain that you would like
+   to get first pass of reviews and explain why achieving ``green`` status for it is not easy/feasible/desired.
+   Similarly if your PR contains ``[WIP]`` in the title or it is marked as ``Draft`` it is not likely to get
+   reviewed by committers unless you specifically ask for it and explain why and what specifically you want
+   to get reviewed before it reaches ``Ready for review`` status. This might happen if you want to get initial
+   feedback on the direction of your PR or if you want to get feedback on the design of your PR.
+
+6. Avoid @-mentioning individual committers in your PR, unless you have good reason to believe that they are
```

Comment:
A general question on the usage of the word `committers` throughout here. Do we think the word `maintainers` could apply here more than `committers`?

Reply:
Good question. We use the term interchangeably - Committer is more formal ASF role, while being maintainer is more GitHub / Generic Open-Source term describing person who has the privilege and responsibility to maintain the repo. In our case the two are the same basically. I attempted to explain the dualism in updated text of the PR and correct "committer" to "maintainer" when it relates to "GitHub" related stuff and "committer" when it relates to more formal role (I think at least I managed to do it). This way we keep both term in use and it maybe even won't look strange.